src/DurableTask.Netherite/StorageLayer/Faster/FaultInjector.cs

```diff
@@ -22,6 +22,7 @@ public enum InjectionMode
         {
             None,
             IncrementSuccessRuns,
+            PermanentFail,
```

Comment:
I think the name should indicate that this flag is injecting specifically on the client start path (unlike the other flags which apply to all calls). Perhaps `FailClientStartup` ?

Reply:
renamed accordingly